pandas/core/indexes/period.py

```diff
@@ -280,6 +295,39 @@ def __new__(
 
         return cls._simple_new(data, name=name, refs=refs)
 
+    @classmethod
+    def from_fields(
```

Comment:
Could you add this method and `from_ordinals` to the API reference?

Reply:
updated+green